test/specs/onboarding.e2e.ts

```diff
@@ -0,0 +1,117 @@
+import path from 'node:path'
+import { browser, expect } from '@wdio/globals'
+import { e2eTestDirPath } from 'test/constants/config'
+import type { ViewSection, Workbench } from 'wdio-vscode-service'
+import { $, cd } from 'zx'
+
+type VsCodeType = typeof import('vscode')
+
+describe('onboarding flow', () => {
+  let workbench: Workbench
+
+  before(async () => {
+    await setUpGitRepo()
+    workbench = await browser.getWorkbench()
+  })
+
+  describe('before the repo is initialized as a radicle repo', () => {
+    it('should load and install the extension', async () => {
+      const extensions = await browser.executeWorkbench(
+        (vscode: VsCodeType) => vscode.extensions.all,
+      )
+      expect(
+        extensions.some((extension) => extension.id === 'radicle-ide-plugins-team.radicle'),
+      ).toBe(true)
+    })
+
+    it('should show the "Radicle" sidebar tab', async () => {
+      const radicleViewControl = await workbench.getActivityBar().getViewControl('Radicle')
+      const title = await radicleViewControl?.getTitle()
+      expect(title).toBe('Radicle')
+    })
+
+    it('should show the `rad init` copy', async () => {
+      await openRadicleSidebarTab(workbench)
+
+      const welcomeText = await findFirstSectionWelcomeText(workbench)
+      expect(welcomeText.some((text) => text.includes('rad init'))).toBe(true)
+    })
+  })
+
+  describe('after the repo has been initialized as a radicle repo', () => {
+    let cliCommandsSection: ViewSection
+
+    before(async () => {
+      await $`rad init --private --default-branch main --name "A_test_blog" --description "Some repo" --no-confirm --verbose`
+      await openRadicleSidebarTab(workbench)
+      const sidebarView = workbench.getSideBar().getContent()
```

Comment:
it's an Activity Bar. It may, or may not be positioned on the side, depending on user preference.